### ISSUE TITLE
collect peers ip address 

### ISSUE THREAD

**Issue by ZigBalthazar** [pactus-project/pactus#631]
## Describe the problem you'd like to have solved

To gather statistics on peer distribution and other related data, it is advisable to collect the IP addresses of peers.
To address this matter, we may utilize the `NotifeeService` events as stated in issue #622 .

**Comment by ZigBalthazar** [comment]
Hint:

```go
func (n *NotifeeService) Connected(_ lp2pnetwork.Network, conn lp2pnetwork.Conn) {
	peerID := conn.RemotePeer()
	peerInfo := n.host.Peerstore().PeerInfo(peerID)
	n.logger.Info("Connected to peer with peerId:" ,"PeerID",peerID)
	n.logger.Info("Connected to peer with MultiAddr:" ,"multiAddrr",peerInfo.Addrs)

	n.eventChannel <- &ConnectEvent{PeerID: peerID}
}
```

**Comment by kehiy** [comment]
@ZigBalthazar 
Hi,
there is 2 project which is showing the node's location in their block explorer:
[sui](https://suiexplorer.com/)
[ontology](https://explorer.ont.io/)

I think they do this with IPs, so you can check how it works for them.
If you find out and get it done (good to provide it in a gRPC API), then I and @Ja7ad can do the same thing in `PacScan`.

**Comment by themantre** [comment]
It is fixed #804
